src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "NETStandard.Library": "1.6.0",
+    "Newtonsoft.Json": "7.0.1",
```

Comment:
If you use Newtonsoft.Json version "9.0.1" or higher, you should be able to get rid of the "imports" in your frameworks below.